db/migrate/20150520144034_add_coupon_to_member.rb

```diff
@@ -0,0 +1,5 @@
+class AddCouponToMember < ActiveRecord::Migration
+  def change
+    add_column :members, :coupon, :string
```

Comment:
It shouldn't affect that tests but why did this change have such a massive effect on the `db/schema.rb` file?